content/sensu-go/5.21/guides/monitor-sensu.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.21"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.21:
+    parent: guides
+---
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+- [Monitor your Embedded etcd](#monitor-your-embedded-etcd)
+- [Monitor your External etcd](#monitor-your-external-etcd)
+- [PostgreSQL PLACEHOLDER](#postgresql-placeholder)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu.
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard.
+
+To completely monitor Sensu (a Sensu backend with internal etcd and an agent), you will need at least one other independent Sensu instance.
+The second Sensu instance will ensure that you are notified when the primary is down and vice versa.
+
+This guide requires Sensu plugins using assets.
+For more information about using Sensu plugins, see [Install plugins with assets][10].
+
+_**NOTE**: This guide describes approaches for monitoring a single backend. The strategies in this guide are also useful for monitoring individual members of a backend cluster._
+
+_**NOTE**: This guide does not go over Sensu agent monitoring. To learn more about monitoring agent state, visit the [keepalive][11] reference_
+
+## Monitor your Sensu backend instances
+
+Monitor the host running the `sensu-backend` in two ways:
+
+* Locally by a `sensu-agent` process for operating system checks and metrics.
+* Remotely from an independent Sensu instance for Sensu components that must be running for Sensu to create events.
+
+### Monitor the Sensu backend locally
+
+Monitor the host that runs `sensu-backend` like any other node in your infrastructure.
```

Comment:
I'm wondering if we need to include service check in combination w/ port checks. Given the issues we've seen (an hopefully fixed) recently where the service was running but internal components weren't, this would make sense to implement in my mind.

However, this opens up a broader conversation about Sensu internals (e.g., the API, authorization component, etc) IMO.